mod-invoice-storage/schemas/voucher.json

```diff
@@ -15,6 +15,12 @@
       "description": "The amount that is actually paid",
       "type": "number"
     },
+    "batchGroupId": {
+      "description": "UUID of the batch group to use when generating batch vouchers",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$",
+      "default": "2a2cb998-1437-41d1-88ad-01930aaeadd5"
+    },
     "batchNumber": {
```

Comment:
As per [wiki](https://wiki.folio.org/pages/viewpage.action?spaceKey=FOLIJET&title=Batch+Voucher+Exports#BatchVoucherExports-invoice), should `batchNumber` field be removed as it is no longer required?

Reply:
Ah, thank you for the reminder... I just added that to the wiki last night after a discussion with Dennis

Reply:
I'll need to update the sample data and example as well.